apis/v1alpha2/validation/httproute_test.go

```diff
@@ -328,6 +330,26 @@ func TestValidateHTTPRoute(t *testing.T) {
 	}, {
 		name:     "valid rewrite path modifier",
 		errCount: 0,
+		rules: []gatewayv1a2.HTTPRouteRule{{
+			Matches: []gatewayv1a2.HTTPRouteMatch{{
+				Path: &gatewayv1a2.HTTPPathMatch{
+					Type:  &pathPrefixMatchType,
+					Value: utilpointer.String("/bar"),
+				},
+			}},
+			Filters: []gatewayv1a2.HTTPRouteFilter{{
+				Type: gatewayv1a2.HTTPRouteFilterURLRewrite,
+				URLRewrite: &gatewayv1a2.HTTPURLRewriteFilter{
+					Path: &gatewayv1a2.HTTPPathModifier{
+						Type:               gatewayv1a2.PrefixMatchHTTPPathModifier,
+						ReplacePrefixMatch: utilpointer.String("foo"),
+					},
+				},
+			}},
+		}},
+	}, {
+		name:     "rewrite path modifier missing path match",
```

Comment:
Maybe add one more test for > 1 path match?

Reply:
+1 from me, that seems like a good one to add.

Reply:
Added, thanks!